exercises11/main.js

```diff
@@ -0,0 +1,124 @@
+// exercise 1: Viết 1 hàm trả về 1 array, nhận vào 2 tham số, tham số thứ 1 là ký tự cần lặp, tham số thứ 2 là số lần lặp
+//cách 1
+function returnArray (charRepeat, numberRepeat) {
+  if (typeof numberRepeat === 'number' && numberRepeat > 0) {
+    const array = []
+    for (let i = 1; i <= numberRepeat; i ++) {
+      array.push(charRepeat)
+    }
+    return array
+  }
+  return 'số lần lặp phải là kiểu số và lớn hơn 0'
+}
+
+// exercise 2
+function reverseArray (array) {
+  if (Array.isArray(array)) {
+    for (let i = 0; i < array.length/2; i ++) {
+      const provisional = array[i]
+      array[i] = array[array.length-1-i]
+      array[array.length-1-i] = provisional
+    }
+    return array
+  }
+  return 'Dữ liệu nhập vào phải là dạng mảng'
+}
+
+// exercise 3
+function deleteElementFalse (array) {
+  if (Array.isArray(array)) {
+    return array.filter(value => Boolean(value));
+  }
+  return 'Dữ liệu nhập vào phải là dạng mảng'
+}
+
+// exercise 4
+function createObject (data) {
+  const obj = {}
+  data.forEach(value => {
+    for (let i = 0; i < value.length/2; i ++) {
+      obj[value[2*i]] = value[2*i+1]
+    }
+  })
+  return obj
+}
+
+// exercise 5
+function sortArray (array) {
+  if (Array.isArray(array)) {
+    for (let i = 0; i < array.length - 1; i ++) {
+      for (let j = i + 1; j < array.length; j++) {
+          if (array[i] > array[j]) {
+            const provisional = array[i]
+            array[i] = array[j]
+            array[j] = provisional      
+          }
+      }
+    }
+    return array
+  }
+  return 'Dữ liệu nhập vào phải là dạng mảng'
+}
+
+// exercise 6
+function checkObject (object) {
+  const check = typeof(object) === 'object' && !Array.isArray(object)? true : false
+  return check
```

Comment:
1 là return như thế này luôn, không cần viết thành 2 lệnh
```suggestion
  return typeof(object) === 'object' && !Array.isArray(object)? true : false
```

2.
```suggestion
  return typeof(object) === 'object' && !Array.isArray(object)
```
Viết thế này là đủ